doc/concepts/pipeline-concepts/pipeline/service.md

```diff
@@ -0,0 +1,41 @@
+# Service
+
+Service is a special type of pipeline that does not process data but provides
+a capability to expose it to the outside world. For example, you can use
+a service to expose a Jupyter™ notebook that has the most
+up-to-date version of your data.
+
+The following pipeline spec extract is an example of how you can expose your
+Jupyter notebook as a service by adding a `service` field:
+
+```json
+{
+    "input": {
+        "pfs": {
+            "glob": "/",
+            "repo": "input"
+        }
+    },
+    "service": {
+        "external_port": 30888,
+        "internal_port": 8888
+    },
+    "transform": {
+        "cmd": [
+            "start-notebook.sh"
+        ],
+        "image": "jupyter/datascience-notebook"
+    }
+}
+```
+
+The service section specifies the following parameters:
+
+| Parameter         | Description   |
+| ----------------- | ------------- |
+| `"internal_port"` | The port to which the code that runs inside the container binds. |
```

Comment:
```suggestion
| `"internal_port"` | The port to which the code that runs inside the container bindings. |
```

Should this be "container bindings"? I'm not actually familiar with this term so check with eng.